rmf_traffic_ros2/include/rmf_traffic_ros2/schedule/Negotiation.hpp

```diff
@@ -124,6 +125,26 @@ class Negotiation
     rmf_traffic::schedule::ParticipantId for_participant,
     std::unique_ptr<rmf_traffic::schedule::Negotiator> negotiator);
 
+  std::shared_ptr<void> register_negotiator(
```

Comment:
Should we add some documentation for this overload? I think the parameter descriptions for the overload below need to be updated too?